package.json

```diff
@@ -34,7 +34,10 @@
     "test:coverage": "npm run test -- --collectCoverage",
     "test:watch": "npm run test -- --watchAll",
     "test:update": "npm run test -- -u",
-    "wait": "wait-on -l -i 1000 tcp:8545 http-get://localhost:3000"
+    "wait": "wait-on -l -i 1000 tcp:8545 http-get://localhost:3000",
+    "next-dev": "next dev",
```

Comment:
end goal is to replace the other scripts right ?

Reply:
yeap this is just for now